static/css/components/merge-request-table.less

```diff
@@ -112,6 +72,106 @@
   }
 }
 
+//NEW STUFF HERE
+
+.dot-wrapper {
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.book-title {
+  font-weight: bold;
+  font-size: 1rem;
+}
+
+.orange-dot {
+  border-radius: 10px;
+  width: 10px;
+  background-color: @orange;
+  height: 10px;
+  margin-right: 12px;
+}
+
+.nav-bar {
+  padding: .5%;
+  background-color: @lightest-grey;
+
+  li a {
+    font-weight: 400;
+    color: @lighter-black;
+  }
+
+  li.selected a {
+    font-size: 14px;
+    color: @orange-five;
+    font-weight: bold;
+    border: none;
+  }
+
+  li:hover a {
+    border: none;
+    font-weight: bold;
+  }
+
+  li:nth-child(n + 3) {
+    float: right;
+  }
+}
+
+.mr-table {
+  width: 100%;
+  background-color: @white;
+
+  tbody {
+    background-color: @white;
+  }
+
+  tr {
+    display: flex;
+    justify-content: space-between;
+    padding: 1%;
+    background-color: @white;
+  }
+
+  tr:nth-child(even) {
+    background-color: @lightest-grey;
+  }
+
+  td {
+    vertical-align: top;
+    padding: 1%;
+  }
+}
+
+.mr-comment-review-cell {
+  display: flex;
+  justify-content: space-between;
+  align-items: start;
+
+  button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    box-shadow: none;
+    width: 65px;
+    height: 50px;
+    margin-right: 40px;
+  }
+
+  &__review {
+    background-color: @primary-blue;
+    border: none;
+    width: 63px;
```

Comment:
Padding for the review button seems off (too much for the top and bottom, but not enough for the sides).  Please refer to Figma for the correct measurements.

Reply:
I took a closer look at the Figma, and the measurements are a bit unclear for the merge request table.  You can see CSS in the right sidebar when an element is selected.  For some reason, the positioning of the button in Figma is `absolute`, so the CSS had `top` and `left` values instead of `padding`.

Anyway, I think that the review button looks much better now!